src/StatsDisplay/StatsDisplay.tsx

```diff
@@ -0,0 +1,183 @@
+import {
+	Container,
+	Table,
+	TableHead,
+	TableRow,
+	TableCell,
+	Box,
+	TableBody,
+	Typography,
+} from '@material-ui/core'
+import React, { FC } from 'react'
+
+interface UserStats {
+	assistsAvg: number
+	assistsTotal: number
+	deathsAvg: number
+	deathsTotal: number
+	gamesCount: number
+	goldEarnedAvg: number
+	goldEarnedTotal: number
+	kDA: number
+	killsAvg: number
+	killsTotal: number
+	timePlayedAvg: number
+	timePlayedTotal: number
+	totalLosses: number
+	totalWins: number
+	winPercentage: number
+}
+
+interface UserStatsProps {
+	userStats: UserStats
+}
+const StatsDisplay: FC<UserStatsProps> = (props) => {
+	const { userStats } = props
```

Comment:
Let's destructure here

```suggestion
const StatsDisplay: FC<UserStatsProps> = ({ userStats }) => {
```